Braintree.xcodeproj/project.pbxproj

```diff
@@ -1747,7 +1754,6 @@
 				BEF177A52B51C7ED004F0F35 /* SEPADebitAccountsRequest.swift */,
 				BEF177A72B51D6A7004F0F35 /* SEPADebitRequest.swift */,
 				BE0B163E27E0CD7C00868C77 /* SEPADirectDebitAPI.swift */,
-				62DE8FBE2B9656BF00F08F53 /* PrivacyInfo.xcprivacy */,
```

Comment:
Is this diff expected? It looks to be removing `PrivacyInfo.xcprivacy` from the SEPA module.

Reply:
ahhh thanks for letting me know! yea it's def a bad merge 😅 - fixed in ba5b597fa1ece265495740a52081676163abbb5d